.all-contributorsrc

```diff
@@ -119,10 +119,10 @@
       ]
     },
     {
-      "login": "yogikhan",
```

Comment:
I'm confused by this change... Are you changing your own entry? Most the time, this file is updated automatically, so there is no need to change it manually.

Reply:
As circle ci was not getting triggered once i raised the PR so i was trying to fix it. So corrected this as well. Yes my username is changed so updated here.